release-notes/3.0/preview/api-diff/preview4/3.0-preview4_System.Diagnostics.md

```diff
@@ -0,0 +1,65 @@
+# System.Diagnostics
+
+``` diff
+ namespace System.Diagnostics {
+     public class Activity {
```

Comment:
Following the recent work by @vancem, this looks good to me.
https://github.com/dotnet/corefx/pull/33207

cc @lmolkova, @SergeyKanzhelev just to review, but looks good (so no action required).